package.json

```diff
@@ -253,6 +263,7 @@
     "test": "npm run compile && node ./node_modules/vscode/bin/test"
   },
   "dependencies": {
+    "simple-git": "^2.20.1",    
```

Comment:
The currently used typescript version is too old for the new package.
#187 should address this or you switch to typescript 3.8.3 with this PR

Reply:
Upgraded typescript in https://github.com/cschlosser/doxdocgen/pull/186/commits/f0a69da5fa72c06679b9feaf3a113842f695a503